assets/scss/_variables_project.scss

```diff
@@ -43,3 +44,4 @@ $otel-registry-license-colors: (
 $primary: map-get($otel-colors, 'blue');
 $secondary: map-get($otel-colors, 'orange');
 $td-enable-google-fonts: false;
+$box-shadow-large: 0 20px 50px rgba(0, 0, 0, 0.8);
```

Comment:
```suggestion
$box-shadow-large: 0 20px 50px rgba(0, 0, 0, 0.8);

```

Reply:
@svrnm, I don't understand this change, from what I'm seeing the suggested change contains the same code.

Please explain further.

Reply:
it adds an empty line to the file which was there before you edited it